### DISCUSSION TITLE
Is it possible to change the font on the fly?

### DISCUSSION THREAD

**Discussion by bend-n** [kovidgoyal/kitty#4946] [Q&A] [answered]
I wish to `echo` output in a different font.

**Comment by page-down** [comment, marked as answer]
Currently kitty cannot change fonts after running, or set fonts for a particular OS window.

There is also no escape sequence that can specify different fonts for certain text in the same window.
It is basically impossible to render two or more monospaced fonts beautifully at the same cell size, since each font's glyphs are not the same size.

**Reply by bend-n** [reply]
Are there other terminals that can change fonts after running?

**Reply by page-down** [reply]
You can reload the configuration after changing the font in `kitty.conf`.

I have not met a terminal that supports `echo` with another font.

**Comment by kovidgoyal** [comment]
No, all the contents of a terminal are typeset in a single font. You can
change that font dynamically, but not change the fonts of parts of the
output.

**Reply by bend-n** [reply]
How does one change the font dynamically for the entire terminal?
